macros/commands/create_elementary_user.sql

```diff
@@ -0,0 +1,34 @@
+{% macro create_elementary_user(user="elementary", password=none, role="ELEMENTARY_ROLE") %}
```

Comment:
1. Why is the role in caps?
2. Any reason not to call the role just `elementary`?

Reply:
from what i gathered when researching, role names are almost always all caps, and usually (thought not always, and this may just be a guild thing) end with the `_ROLE` suffix

Reply:
Cool. We'll learn if that's not the case.